Gemfile

```diff
@@ -44,19 +44,23 @@ gem 'slack-notifier', '~> 1.0.0'  # SlackAgent
 gem 'hypdf', '~> 1.0.10'          # PDFInfoAgent
 
 # Weibo Agents
-# FIXME needs to loosen omniauth dependency
-gem 'weibo_2', github: 'dsander/weibo_2', branch: 'master'
+# FIXME needs to loosen omniauth dependency, add rest-client
+gem 'weibo_2', github: 'albertsun/weibo_2', branch: 'master'
 
 # GoogleCalendarPublishAgent and GoogleTranslateAgent
-gem 'google-api-client', '~> 0.7.1', require: 'google/api_client'
+gem 'google-api-client', '~> 0.13'
+# gem 'google-api-client', '~> 0.7.0' #, require: 'google/api_client'
+gem 'google-cloud-translate', '~> 1.0.0', require: 'google/cloud/translate'
+# gem 'google-cloud-bigquery', '~> 0.27.0', require: 'google/cloud/bigquery'
```

Comment:
Yea, can take these out. Unless people would be interested in having the bigquery agent in core.